katsdpimager/fast_math.py

```diff
@@ -0,0 +1,21 @@
+"""Variants of standard maths functions written for speed."""
+
+import numpy as np
+import numba
+
+
+@numba.vectorize([numba.complex64(numba.float32),
```

Comment:
What is the speedup of Numba vs the original?

Reply:
Looks like 40ns -> 25ns per element on my machine.